vm/base-context.cpp

```diff
@@ -236,3 +236,14 @@ BasePluginContext::BlamePluginError(SourcePawn::IPluginFunction *pf, const char
   va_end(ap);
   return 0;
 }
+
+IFrameIterator *BasePluginContext::GetFrameIterator()
+{
+  FrameIterator *it = new FrameIterator();
+  return it;
+}
+
+void BasePluginContext::FreeFrameIterator(IFrameIterator *it)
+{
+  delete it;
```

Comment:
`IFrameIterator` does not have a virtual destructor, so you have to `static_cast<FrameIterator*>(it)` to delete it.